connexion/operations/openapi.py

```diff
@@ -275,7 +275,15 @@ def _get_body_argument(self, body, arguments, has_kwargs, sanitize):
         if len(arguments) <= 0 and not has_kwargs:
             return {}
 
-        x_body_name = sanitize(self.body_schema.get('x-body-name', 'body'))
+        # prefer the x-body-name as an extension of requestBody
+        x_body_name = sanitize(self.request_body.get('x-body-name', None))
+
+        if not x_body_name:
+            # x-body-name also accepted in the schema field for legacy connexion compat
+            logger.warning('x-body-name within the requestBody schema will be deprecated in the '
```

Comment:
No expert, but I think a DeprecationWarning might be the most pythonic option here?
https://docs.python.org/3/library/warnings.html#warnings.warn

```
warnings.warn(message, DeprecationWarning, stacklevel=2)
```

Reply:
Actually what I wanted to do :see_no_evil: